package.json

```diff
@@ -38,38 +41,38 @@
     "argv-parse": "^1.0.1",
     "babel-eslint": "^10.0.0",
     "babel-loader": "^8.0.0",
-    "compression-webpack-plugin": "~1.1.11",
     "cross-env": "^5.2.0",
-    "css-loader": "^0.23.1",
+    "css-loader": "4.3.0",
+    "css-minimizer-webpack-plugin": "^4.2.2",
     "cssnano": "^4.1.10",
     "dotenv": "^5.0.0",
     "eslint": "^6.7.2",
     "eslint-plugin-spellcheck": "0.0.17",
-    "expose-loader": "~0.6.0",
-    "extract-text-webpack-plugin": "^3.0.0",
-    "file-loader": "^0.9.0",
     "graphql": "^15.5.0",
     "highlight.js": "~9.14.0",
-    "node-sass": "^4.5.0",
-    "optimize-css-assets-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "2.7.5",
+    "node-sass": "6.0.0",
```

Comment:
Is this an update we can do independent of the webpack change?

Reply:
no - `sass-loader` only supports `node-sass` v6 from `sass-loader` v11.1.0
and `sass-loader` v11.0.0 only supports wepack 5